test/models.test.js

```diff
@@ -50,10 +52,74 @@ describe('collections', function () {
 
 describe('models', function() {
   describe('centre', function() {
+    beforeEach(function() {
+      var io;
+
+      this.server = new socket();
+      this.fakeio = function(server) {
+        return function() {
+          return server.socketClient;
+        }
+      };
+
+      io = sailsIOClient(this.fakeio(this.server));
+      io.sails.autoConnect = false;
+
+      this.socket0 = io.sails.connect();
+    });
     it('should be instantiated', function() {
       var model = new models.Centre();
 
       expect(model).not.to.be(undefined);
```

Comment:
in general unit tests should have one assert.
you should also return the expect so that mocha picks it all up explicity